benchmarks/entropy_adiabat/plugins/entropy_model.cc

```diff
@@ -47,10 +99,11 @@ namespace aspect
       material_lookup = std::make_unique<Utilities::StructuredDataLookup<2>>(7,1.0);
       material_lookup->load_file(data_directory+material_file_name,
                                  this->get_mpi_communicator());
+      lateral_viscosity_prefactor_lookup = std::make_unique<internal::LateralViscosityLookup>(data_directory+lateral_viscosity_prefactor_file_name,
+                                           this->get_mpi_communicator());
     }
 
 
-
```

Comment:
Keep this empty line. In fact our convention is to have three empty lines between functions in .cc files (and one empty line in .h files).